translations/hu/get5.phrases.txt

```diff
@@ -212,6 +220,10 @@
     {
         "hu"            "Egy admin, szüneteli a mérkőzést."
     }
+    "AdminForceEndWithWinnerInfoMessage"
+    {
+        "hu"            "Egy admin véget vetett a mérkőzésnek és a {1} csapatot állította be győztesnek."
+    }
     "AdminForceUnPauseInfoMessage"
     {
         "hu"            "Egy admin, a mérkőzés folytatását hajtotta végre."
```

Comment:
`Egy admin hajtotta végre a mérkőzés folytatását.`

Reply:
Elfogadva a javaslat. The suggestion is accepted